src/ui_basic/editbox.h

```diff
@@ -104,6 +104,9 @@ struct EditBox : public Panel {
 	void calculate_selection_boundaries(uint32_t& start, uint32_t& end);
 	void delete_selected_text();
 	void copy_selected_text();
+	void set_caret_to_cursor_pos(int32_t x);
```

Comment:
Plase add some comments, epecially about the context of x, index and text_w

Reply:
changed the names. descriptive code > comments :)